### ISSUE TITLE
Cypress runner closes on clicking Runs tab on Test runner

### ISSUE THREAD

**Issue by Nellolilu** [cypress-io/cypress#19437]
### Current behavior

Hi,

Faced same Issue as mentioned here, but newer version: https://github.com/cypress-io/cypress/issues/18150

I was trying to setup Cypress Dashboard using my google account.
Setup completed and I could see the dashboard updated
But when I click Runs tab on the Test Runner, it crashes and the test runner unexpectedly crashes with the following error:

**ERROR:**
```
TypeError: openProject.getRuns(...).then is not a function
    at IpcMainImpl.handleEvent (/Users/nelli.neb/Library/Caches/Cypress/8.4.0/Cypress.app/Contents/Resources/app/packages/server/lib/gui/events.js:367:8)
    at apply (/Users/nelli.neb/Library/Caches/Cypress/8.4.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/lodash/lodash.js:492:17)
    at IpcMainImpl.wrapper (/Users/nelli.neb/Library/Caches/Cypress/8.4.0/Cypress.app/Contents/Resources/app/packages/server/node_modules/lodash/lodash.js:5379:16)
    at IpcMainImpl.emit (events.js:315:20)
    at Object.<anonymous> (electron/js2c/browser_init.js:161:9469)
    at Object.emit (events.js:315:20)
```

### Desired behavior

The Cypress TestRunner should stay open, recording should be possible

### Test code to reproduce

Preconditions: 
- projectId from Cypress Dashboard implemented in cypress.json
- testrunner opened

Steps to reproduce:
- click on "Runs" Tab in Testrunner

Behavior:
- Testrunner closed
- Terminal throws error

### Cypress Version

9.1.0

**Comment by jennifer-shehane** [comment]
The error message shows that you are running the cached 8.4.0 version of Cypress. This issn't was fixed in 8.4.1. Please ensure you have the latest Cypress installed and are running that version.

**Comment by vasslaci** [comment]
I also get the error with the latest version.

Cypress:        9.2.0
Browser:        Electron 94 (headless) 
Node Version:   v14.13.1 (/var/lib/jenkins/tools/jenkins.plugins.nodejs.tools.NodeJSInstallation/nodejs/bin/node)

...
...
...

	✓ Oldalsó menüsor - Költséghely adatok (28447ms)
	1) Oldalsó menüsor - Költségviselő adatok
 
The Test Runner unexpectedly exited via a exit event with signal SIGSEGV
Please search Cypress documentation for possible solutions:
 
https://on.cypress.io
 
Check if there is a GitHub issue describing this crash:
 
https://github.com/cypress-io/cypress/issues
 
Consider opening a new issue.
 
----------
 
Platform: linux-x64 (Debian - 10)
Cypress Version: 9.2.0
Build step 'Execute shell' marked build as failure
Archiving artifacts
Finished: FAILURE
